content/python/concepts/strings/terms/lstrip/lstrip.md

```diff
@@ -0,0 +1,60 @@
+---
+Title: 'lstrip'
+Description: 'Removes leading characters from a string'
+Subjects:
+  - 'Computer Science'
+  - 'Code Foundations'
+Tags:
+  - 'Strings'
+  - 'Methods'
+  - 'Functions'
+CatalogContent:
+  - 'https://www.codecademy.com/learn/learn-python-3'
+  - 'https://www.codecademy.com/learn/paths/computer-science'
+---
+
+The **`.lstrip()`** method removes the leading characters from a string
+
+## Syntax
+```pseudo
+string.lstrip(chars)
+```
+The `.lstrip()` method defaultly removes the whitespace at the beginning of a string. The optional `chars` parameter is a string that allows you to specify leading characters to be removed.
+
+## Example
+Unless `chars` is specified, `.lstrip()` removes the leading whitespace:
```

Comment:
it may be more concise to put all three examples in one code block. And generally we prefer to use a ```shell block for the output instead of using comments.

Reply:
If I put all the examples in one code block how should I designate which output in the ```shell block is from which print statement?

Reply:
They should be matched by the order